bemani/backend/jubeat/clan.py

```diff
@@ -1103,8 +1093,14 @@ def handle_gameend_final_request(self, request: Node) -> Node:
 
         return Node.void('gameend')
 
-    def format_scores(self, userid: UserID, profile: Profile, scores: List[Score]) -> Node:
-
+    def format_scores(self, userid: UserID, profile: Profile, scores: List[Score], mdata_ver: Optional[int]) -> Node:
+        if mdata_ver is None:
+            mdata_ver = 1
```

Comment:
Why is this optional then? Its confusing that the function would take an optional argument and if its not specified pick one instead of returning all music. I would default this to selecting from all music if mdata_ver is none here, not arbitrarily choosing the first one.